buildenv

```diff
@@ -31,11 +31,82 @@ zopen_check_results()
 zopen_append_to_env()
 {
   # echo envars outside of PATH, MANPATH, LIBPATH
+
+  # i'm not seeing these prints in zopen build.
+  # perhaps its written the setup.sh in the 
+  # usr/local/zopen dir
+
+  echo "***** in append_to_env() start"
+  echo "CWD: "
+  pwd
+
+
+
+  # update the path for depot_tools
+  export PATH=${PATH}:${HOME}/zopen/dev/depot_toolsport/depot_tools
+  # update the path for pip
+  export PATH=${PATH}:${HOME}/.local/bin
+  # update the compiler settings
+  # TODO: this can probably be moved to v8 build later
+  export CC=clang
+  export CXX=clang++
+  export LINK=clang++
+  # Disable updating depot_tools
+  export DEPOT_TOOLS_UPDATE=0
+
+  # Disable commands:
+  # * ensure_bootstrap
+  # * update_depot_tools
+  export DEPOT_TOOLS_BOOTSTRAP_PYTHON3=0
+
+  # Using depot_tools disable metrics.
+  # Comment out this line if you wish to participate.
+  #gclient metrics --opt-out
+
+
+  echo "***** in append_to_env() end"
 }
 
+
+# Notes:
+# These steps create this directory 
+# /z/jd895801/zopen/usr/local/zopen/depot_tools
+# during the `zopen build` process.
+#
+# This code is run when zopen build is issued
+# and the CWD is ~/zopen/dev/depot_toolsport
+#
+# Example buildenvs which use this portion of 
+# code are:
+#
+# https://github.com/ZOSOpenTools/sudoport/blob/main/buildenv#L49
+#
+# https://github.com/ZOSOpenTools/gpgport/blob/main/buildenv#L50
+
 zopen_append_to_setup()
 {
-  # echo commands that will run when installing via setup.sh
+# echo commands that will run when installing via setup.sh
+cat <<EOF
+echo "***** append_to_setup() start"
+echo "***** append_to_setup() start"
+echo "***** append_to_setup() start"
+echo "CWD:"
+pwd
+# /z/jd895801/zopen/usr/local/zopen/depot_tools/depot_tools-main
+echo "ZOPEN_INSTALL_DIR: $ZOPEN_INSTALL_DIR"
+# /z/jd895801/zopen/usr/local/zopen/depot_tools/depot_tools-main
+echo "Creating a python venv..."
+python3 -m venv venv
```

Comment:
Same for this, for "python3 -m venv venv" to show up in the setup.sh script, you'll need to echo it:
```
echo "python3 -m venv venv"
```